womenwhogo.org/index.html

```diff
@@ -788,6 +788,23 @@ <h1 class="text-center">Women Who Go</h1>
           </p>
         </div>
       </div>
+      <div class="row chapter">
+        <div class="col-xs-5">
+          <p class="lead">Utah:</p>
+        </div>
+        <div class="col-xs-7">
+          <p class="col-xs-2 text-center link">
+            <a href="https://www.meetup.com/es/Women-Who-Go-Utah" target="_blank">
```

Comment:
This link is for the Spanish translation of the site. I just wanted to confirm quick that you wanted to link to the Spanish translation instead of the default English one?

https://www.meetup.com/Women-Who-Go-Utah/

Reply:
sorry, copy/past failed me this time. I just pushed up a change that should have it as english